go/kbfs/simplefs/simplefs.go

```diff
@@ -3032,30 +3078,17 @@ func (k *SimpleFS) SimpleFSGetGUIFileContext(ctx context.Context,
 	if err != nil {
 		return keybase1.GUIFileContext{}, err
 	}
-
-	var encodedSegments []string
-	for _, segment := range strings.Split(path.Path, "/") {
+	encodedSegments := []string{"/files"}
```

Comment:
Minor optimization: split `kbfsPath` above this, and calculate the exact length of the `encodedSegments` slice before you allocate it.